clients/src/main/java/org/apache/kafka/common/requests/ApiVersionsRequest.java

```diff
@@ -30,8 +30,14 @@
     /* v1 request is the same as v0. Throttle time has been added to response */
     private static final Schema API_VERSIONS_REQUEST_V1 = API_VERSIONS_REQUEST_V0;
 
+    /**
+     * The version number is bumped to indicate that on quota violation brokers send out responses before throttling
+     * (KIP-219).
+     */
+    private static final Schema API_VERSIONS_REQUEST_V2 = API_VERSIONS_REQUEST_V1;
+
     public static Schema[] schemaVersions() {
-        return new Schema[]{API_VERSIONS_REQUEST_V0, API_VERSIONS_REQUEST_V1};
+        return new Schema[]{API_VERSIONS_REQUEST_V0, API_VERSIONS_REQUEST_V1, API_VERSIONS_REQUEST_V2};
```

Comment:
@ijuma had a comment on the KIP discuss thread about the approach to update only ApiVersions version. Can you respond on the thread, please?

Reply:
Yes, I replied to the vote thread for seeking comments. Could you also respond to that thread since you were one of the original voters?